packages/terra-infinite-list/src/InfiniteList.jsx

```diff
@@ -11,6 +11,10 @@ import styles from './InfiniteList.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * String that labels the modal for screen readers.
+   */
+  ariaLabel: PropTypes.string.isRequired,
```

Comment:
making this required makes this a MVB. @ryanthemanuel do we want to make this a MVB or just add the prop so teams have the ability to make this accessible?

Reply:
Keep this minor.

Reply:
Curious to know why do we need this prop for adding aria-label to infinite-list.
When we can directly add 'aria-label' as custom prop to `infinite-list`  like : 
```javascript
  <InfiniteList
    role="listbox"
    aria-label="label"
  >
```
by Updating [List](https://github.com/cerner/terra-framework/blob/d660b07885ef6d8389390c00f057b4194f9ea135/packages/terra-infinite-list/src/InfiniteList.jsx#L525) tag in render of InfiniteList.jsx to have `{...customProps} ` like : 
```Javascript
<List {...customProps} {...attrSpread} refCallback={refCallback}>
```

Reply:
By making it a prop we can make it required.

Reply:
We like to call out props that are more first class concepts (even if they can be passed in).  It makes the API a little more clearer.  In the future, we could also make this required if we wanted.